helpers/dbload.ts

```diff
@@ -107,4 +112,4 @@ export default {
   sequelize
 }
 
-export { User, UserLesson }
+export { User, UserLesson, options }
```

Comment:
Do we plan to use the `options` object in the future, or in other places in the codebase? I'm not able to think of a scenario where we would use the `options` object in another file. Seems to me that it is a private variable whose only purpose is to be used as the third parameter when calling the Sequelize function in this file, so exporting 'options' makes things more confusing. It makes me think that the variable is meant to be used in other places in the codebase

Reply:
It won't be used in other places at all, I added it as export for easier testing. Otherwise I am not sure how to write tests for dbload without explicitly creating import. 
I could rename it to `testOptions` or something like that.

Reply:
I see, thanks for explaining! I could not think of a better way to write the tests either.